java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -3045,11 +3045,13 @@ void testWindowingCollectList() {
 
   @Test
   void testWindowingCollectSet() {
-    // TODO: Add test cases for aggCollectWithUnEqNulls and aggCollectWithEqNaNs after
-    // the issue (https://github.com/rapidsai/cudf/issues/8405) being addressed.
```

Comment:
For the future, I'm not a fan of lumping a bunch of unrelated changes into a PR.  These two changes are completely orthogonal and should have been done as separate PRs.  It's not like the overhead of a separate PR is _that_ high, especially since the Java PRs are skipping CI.

Reply:
Sorry, I will be aware of this in next time.